priv/templates/extended_bin

```diff
@@ -271,7 +271,7 @@ relx_rem_sh() {
     # if it is then we must generate a node name to use for the remote node
     if [  "11.0" = "$(printf "%s\n11.0" "${ERTS_VSN}" | sort -V | head -n1)" ] ; then
```

Comment:
will this break on 12.0?

Reply:
Na. It adds `11.0` to the list of versions with the current erts vsn, then sorts and sees if the sorted versions has `11.0` first. If it doesn't that means we are earlier than 11.0 and if it does that means we are at 11.0 or later.